spec/factories/nufs_accounts.rb

```diff
@@ -7,10 +12,19 @@
   expires_at { Time.zone.now + 1.month }
   created_by 0
 
+  # TODO: add the owner as part of the default so the factory is valid.
+  trait :with_owner do
```

Comment:
Take a look at https://github.com/tablexi/nucore-nu/pull/104/files I cherry-picked up the commit (5363f3ee15f567729dfdf47e9616778c6a720fa6) that was relevant to open. This needs to be outside the `overridable_factory` section, otherwise it'll cause problem downstream.